packages/remediations/src/newSteps/reviewActions.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState, Fragment, useEffect } from 'react';
+import propTypes from 'prop-types';
+import useFieldApi from '@data-driven-forms/react-form-renderer/dist/esm/use-field-api';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/esm/use-form-api';
+import './reviewActions.scss';
+import {
+    Radio,
+    Text,
+    TextContent,
+    Stack,
+    StackItem
+} from '@patternfly/react-core';
+import { Table, TableVariant, TableHeader, TableBody, sortable } from '@patternfly/react-table';
+import { CloseIcon, RedoIcon } from '@patternfly/react-icons';
+
+const ReviewActions = (props) => {
+    const { issues, issuesMultiple } = props;
+    const { input } = useFieldApi(props);
+    const formOptions = useFormApi();
+    const [ sortByState, setSortByState ] = useState({ index: undefined, direction: undefined });
+
+    useEffect(() => {
+        formOptions.getState().values['manual-resolution'] === undefined && formOptions.change('manual-resolution', true);
```

Comment:
Can you use `initialValue` instead?

Reply:
I wanted to avoid the action substeps being visible before I actually select that I want to review them. Maybe they should be visible...